README.md

```diff
@@ -92,6 +92,30 @@ http-server --proxy http://localhost:8080?
 
 Note the `?` at the end of the proxy URL. Thanks to [@houston3](https://github.com/houston3) for this clever hack!
 
+## Using SSL
```

Comment:
To be pedantic, I think we should say "TLS/SSL" here, since the user will technically be enabling TLS, even though people sometimes call it SSL.

Reply:
OK